application/config/constants.php

```diff
@@ -83,40 +83,3 @@
 defined('EXIT_DATABASE')       OR define('EXIT_DATABASE', 8); // database error
 defined('EXIT__AUTO_MIN')      OR define('EXIT__AUTO_MIN', 9); // lowest automatically-assigned error code
 defined('EXIT__AUTO_MAX')      OR define('EXIT__AUTO_MAX', 125); // highest automatically-assigned error code
-
-/*
-|--------------------------------------------------------------------------
-| Custom Defined Constants
-|--------------------------------------------------------------------------
-|
-*/
-
-/**
- * The event registration sheet we are storing our information on (i.e. adhoc database).
- */
-const REGISTRATION_SPREADSHEET_ID = '1YvLmK7DbVuf5O3UDKlYaNOH1Zd8ohg_eEibzwfh2IME';
-
-/**
- * Stripe public and private keys for development.
- */
-if (!defined('STRIPE_PUBLIC_KEY')) {
-    define('STRIPE_PUBLIC_KEY', 'pk_test_A4wjqVPPn530rgAXv6sHKgSl00opCMVX9A');
-    define('STRIPE_PRIVATE_KEY', 'sk_test_OMC00A11yJUakUU4kx6KoGTp0028EYnLBa');
-}
-
-if (!defined('MAIL_AUTH_EMAIL')) {
-    $json = file_get_contents("private_keys/auth_props.json");
-    $authProps = json_decode($json, true);
-
-    define('MAIL_AUTH_EMAIL', $authProps["mail.email"]);
-    define('MAIL_AUTH_PASSWORD', $authProps["mail.password"]);
-}
-
-/**
- * Google sheet ID and sheet name for ASPA's membership spreadsheet.
- */
-const MEMBERSHIP_SPREADSHEET_ID = '10mwPhiOR_Vfsfw8WHereu4Y5KOsuSWkJFGhrf6Mfk9I';
-const MEMBERSHIP_SHEET_NAME = 'Sheet1';
-
-// Feature Toggles
-const CHECK_MEMBERSHIP_PAYMENT = FALSE;
```

Comment:
This has to stay in the `constants.php` file (as we want it to apply on both production and dev). Otherwise, you can also add this field into the `production/constants.php` file.